<launch>
	<node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk_node" output="screen">
		<!-- node parameters -->
		<param name="serial_name" type="string" value="/dev/cu.SLAB_USBtoUART"/>
		<param name="baud_rate" type="int" value="230400"/>
		<param name="app_id" type="int" value="10086"/>
		<param name="app_sdk_level" type="int" value="2"/>
		<param name="app_version" type="int" value="1"/>
		<param name="app_bundle_id" type="string" value="12345678901234567890123456789012"/>
		<param name="enc_key" type="string" value="DJI-DEMO AES256 KEY-lala-haha-MA"/>	
		<param name="mavlink_ip" type = "string" value = "127.0.0.1" />
		<param name="mavlink_port" type = "int" value = "14550" />
	</node>
</launch>
